### PR TITLE
Add MPL

### PR DESCRIPTION
**Pull request by carlbordum**
Mozilla Public License 2.0 Source Code Form License Notice has been
added to most or all files containing source code.

	hurray for open source

### CODE REVIEW

File: agentbase/java/src/main/java/dk/magenta/mox/agent/DummyMessageGenerator.java [hunk 1]
```diff
@@ -1,3 +1,13 @@
+/*
+Copyright (C) 2015-2019 Magenta ApS, http://magenta.dk.
```
Comment (danchr) [inline agentbase/java/src/main/java/dk/magenta/mox/agent/DummyMessageGenerator.java]: Customarily, copyright headers use something like this:

```java
/*
 * Copyright…
 *
```

…and so on. At least that's what we did in os2mo 🙂 There's a test that verifies it in `test_style.py` actually…

File: agents/MoxRestFrontend/src/main/java/dk/magenta/mox/moxrestfrontend/RestMessageHandler.java [hunk 2]
```diff
@@ -1,133 +1,10 @@
-package dk.magenta.mox.moxrestfrontend;
```
Comment (danchr) [inline agents/MoxRestFrontend/src/main/java/dk/magenta/mox/moxrestfrontend/RestMessageHandler.java]: Intentional?

File: agentbase/java/src/main/java/dk/magenta/mox/agent/DummyMessageGenerator.java (hunk 1 above)
Comment (carlbordum) [reply agentbase/java/src/main/java/dk/magenta/mox/agent/DummyMessageGenerator.java]: What? I took this from the files in `db/templates/`

File: oio_rest/guniconfig.py [hunk 3]
```diff
@@ -1,3 +1,11 @@
+# Copyright (C) 2015-2019 Magenta ApS, http://magenta.dk.
```
Comment (danchr) [inline oio_rest/guniconfig.py]: This is a configuration file; we shouldn't assert our rights in it.

File: oio_rest/oio_rest/db_templating.py [hunk 4]
```diff
@@ -1,3 +1,11 @@
+# Copyright (C) 2015-2019 Magenta ApS, http://magenta.dk.
+# Contact: info@magenta.dk.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
 #!/usr/bin/env python3
```
Comment (danchr) [inline oio_rest/oio_rest/db_templating.py]: She-bangs _must_ be on the first line.

File: agentbase/java/src/main/java/dk/magenta/mox/agent/DummyMessageGenerator.java (hunk 1 above)
Comment (danchr) [reply agentbase/java/src/main/java/dk/magenta/mox/agent/DummyMessageGenerator.java]: Well, they're SQL Jinja2 templates — this is Java 🙂

Just trying to point what I've usually been seeing; it's not all [that](http://red.bikeshed.com) important.

File: oio_rest/oio_rest/db_templating.py (hunk 4 above)
Comment (carlbordum) [reply oio_rest/oio_rest/db_templating.py]: damn, but that file isnt executable

Comment (danchr) [reply oio_rest/oio_rest/db_templating.py]: Well, it sort of is for development purposes, but probably only as a module. So:
```suggestion
```

File: agents/MoxRestFrontend/src/main/java/dk/magenta/mox/moxrestfrontend/RestMessageHandler.java (hunk 2 above)
Comment (carlbordum) [reply agents/MoxRestFrontend/src/main/java/dk/magenta/mox/moxrestfrontend/RestMessageHandler.java]: No thats bad!

**Comment by agger-magenta** [comment]
Detalje, men:

```
sed -i "s#http://magenta.dk#https://magenta.dk#" $(find . -name '*.py' -o -name '*.java' -o -name '*.sql' -o -name '*.sh')
```
???

**Comment by agger-magenta** [comment]
Ellers ser det helt fint ud.

File: agents/MoxRestFrontend/src/main/java/dk/magenta/mox/moxrestfrontend/RestMessageHandler.java [hunk 5]
```diff
@@ -8,3 +8,4 @@
 */
 
 
+package dk.magenta.mox.moxrestfrontend;
```
Comment (danchr) [inline agents/MoxRestFrontend/src/main/java/dk/magenta/mox/moxrestfrontend/RestMessageHandler.java]: Erm, you deleted the rest of the file as well, right?

File: python_agents/mox_elk_log.py [hunk 6]
```diff
@@ -1,3 +1,11 @@
+# Copyright (C) 2015-2019 Magenta ApS, https://magenta.dk.
+# Contact: info@magenta.dk.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
 #!%PYTHON%
```
Comment (danchr) [inline python_agents/mox_elk_log.py]: This is a shebang, of sorts — probably for templating, somehow. Should probably be:

```python
#!/usr/bin/env python3
```

(Assuming this script works in Python 3.)

Comment (agger-magenta) [reply python_agents/mox_elk_log.py]: That's a Python 2 script, though.

Comment (agger-magenta) [reply python_agents/mox_elk_log.py]: But it _might_ work in Python 3.

Comment (carlbordum) [reply python_agents/mox_elk_log.py]: It doesn't work in Python3 and the file isn't executable anyway (rendering a shebang useless).

I'm leaving it as-is, unless anyone feels strongly about how exactly it should be

**Pull request merged**